guides/createFractionalSaleMakerAsk.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Selling part of a fraction"
+group: Guides
+---
+
+> [!WARNING]
+>
+> These code snippets are just examples and the data should never be used as is
+
+# How to create a fractional sale
+
+A fractional sale allows users to buy a specified number of units from a fraction. The seller can define a minimum and maximum number of units to sell per transaction.
+
+The code snippet below is an example of how to create a maker ask using the `@hypercerts-org/marketplace-sdk` library.
+
+The main steps are:
+
+1. Initialize a HypercertExchangeClient class instance by providing the chain id, [RPC provider](https://docs.ethers.io/v6/api/providers/) and a [signer](https://docs.ethers.org/v6/api/providers/#Signer).
+2. Use the `createFractionalSaleMakerAsk` method to create a maker ask, that allows for selling parts of a fraction.
+3. Check and grant necessary approvals for transferring assets.
+4. Sign the maker ask order with `signMakerOrder` method.
+
+Here is an example:
+
+```ts
+import { parseEther } from "ethers";
+import { HypercertExchangeClient, ChainId, CollectionType, StrategyType } from "@hypercerts-org/marketplace-sdk";
+
+const hypercertExchangeClient = new HypercertExchangeClient(ChainId.OPTIMISM, provider, signer);
+
+// The ID of the fraction that will be put on sale
+const fractionId = 13601086205829910384631083059047775411896320n;
+
+// Create the fractional sale order for a hypercert
+// This will also take care of fetching and setting the order nonce correctly
+const order = await hypercertExchangeClient.createFractionalSaleMakerAsk({
+  startTime: Date.now() / 1000, // Use it to create an order that will be valid in the future (Optional, Default to now)
```

Comment:
We discussed yesterday that that would fail right? Taking blocktimes and on chain validations into account?

Reply:
Fixed, made it 5 minutes into the future now.